ckanext/canada/templates/public/snippets/organization.html

```diff
@@ -25,22 +25,24 @@
     {% endif %}
 
     {% if show_nums %}
-    <div class="row">
-      <div class="col-md-6">
-        <h4 class="text-center">{{ _('Members') }}</h4>
-        <p class="font-xlarge text-center">
-          {{ h.SI_number_span(h.organization_member_count(organization.name)) }}
-        </p>
-      </div>
-      <div class="col-md-6">
-        <h4 class="text-center">{{ _('Datasets') }}</h4>
-
-        <p class="font-xlarge text-center">
-          {{ h.SI_number_span(organization.package_count) }}
-        </p>
-      </div>
-    </div>
-  {% endif %}
+      {% set member_count = h.organization_member_count(organization.name) %}
```

Comment:
This feels like a really inefficient way to get this data, and lots of work to remove orgs that fail the test. Can we have the list filtered and the number of members returned by the organization search action instead?

Reply:
@wardi this snippet is used in `read_base` of the organization templates. So it would only be rendered on the `organization_read` type stuff, not in any of the lists.

We did it this way because getting the users back from group/org show is trapped behind the `ckan.auth.public_user_details` config option

Reply:
@wardi HOWEVER https://github.com/ckan/ckan/pull/7007 there was this PR I did that was accepted into master which just allowed all of the group and org members to be public.

Backporting this would allow us to easily display the package counts and member counts on orgs

Reply:
I'm fine with backporting your upstream change. It simplifies our templates now and will go away when we upgrade in the future